docs/src/modules/java/pages/author-your-first-service.adoc

```diff
@@ -49,8 +48,8 @@ Windows 10+::
 [source,command line, subs="attributes"]
 ----
 mvn archetype:generate ^
-  -DarchetypeGroupId=io.akka ^
-  -DarchetypeArtifactId=akka-javasdk-archetype ^
+  -DarchetypeGroupId=com.example ^
+  -DarchetypeArtifactId=helloworld ^
```

Comment:
this must be wrong, these are the coordinates of the akka-javasdk-archetype, not the generated project